build.gradle

```diff
@@ -247,6 +247,8 @@ dependencies {
     exclude group: 'com.github.tomakehurst', module: 'wiremock-standalone'
   }
 
+  testImplementation group: 'com.jayway.awaitility', name: 'awaitility', version: '1.7.0'
```

Comment:
do we need it under test scope?

Reply:
I was thinking this could be reused by integration tests maybe. A bit speculative, can move it straight to functional.

Reply:
done